common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -114,6 +114,8 @@ public enum Feature {
 
         OID4VC_VCI("Support for the OID4VCI protocol as part of OID4VC.", Type.EXPERIMENTAL),
 
+        SCIM("Synchronise users and groups with registered SCIM endpoints", Type.EXPERIMENTAL),
```

Comment:
Can we have a name that makes it clear that we are not really delivering SCIM but a part of it? In this case, we are adding a SCIM-compliant user storage so perhaps something like `scim-federation` or something ...

Reply:
Sorry, a new commit sounds good. We can squash them all before merging.